segmentation-templates-extension/src/SegmentationTemplate.liquid

```diff
@@ -0,0 +1,52 @@
+{%- if flavor contains "react" -%}
+import {
+  reactExtension,
+  useApi,
+  CustomerSegmentationTemplate,
+} from '@shopify/ui-extensions-react/admin';
+
+// The target used here must match the target used in the extension's toml file (./shopify.extension.toml)
+const TARGET = 'admin.customers.segmentation-templates.render';
+
+export default reactExtension(TARGET, () => <App />);
+
+function App() {
+  // The useApi hook provides access to several useful APIs like i18n.
+  const {i18n} = useApi(TARGET);
+
+  const templateQuery = 'number_of_orders = 1 AND products_purchased(id: (product_id)) = true'
+  const templateQueryToInsert = 'number_of_orders = 1 AND products_purchased(id: (',
+
+  return (
+    // The CustomerSegmentationTemplate component provides an API for setting the title, description, icon, query, and query to insert to the segmentation template.
+    <CustomerSegmentationTemplate
+        title={i18n.translate("title")}
```

Comment:
Do we need a separate `title` field? Names are now automatically localized so we don't really need devs to manually set the title

Reply:
Let me know and I'll make the changes, just want to make sure I am using the fields as intended.

Reply:
So that would mean one template per extension?

Reply:
Sorry, I want to clarify that `description` isn't currently supported yet but we plan to add support for it very soon. There's a few PRs in the works

Reply:
We would need to expose it - it doesn't currently exist right now because there's been no need to use it